packages/jetpack-logo/src/Logo.php

```diff
@@ -0,0 +1,67 @@
+<?php
+/**
+ * A logo for Jetpack.
+ *
+ * @package jetpack-logo
+ */
+
+namespace Jetpack\Assets;
+
+/**
+ * Create and render a Jetpack logo.
+ */
+class Logo implements Renderable_Interface {
+
+	private $assets_manager;
+	/**
+	 * Absolute URL of the Jetpack logo.
+	 *
+	 * @var string
+	 */
+	private $url;
+
+	/**
+	 * Default image url
+	 *
+	 * @var string
+	 */
+	private $default_filename = 'logo.svg';
+
+	public function __construct( $assets_manager = null ) {
+		$this->assets_manager =
```

Comment:
I like the dependency injections you're suggesting. They make everything way easier to test.